code/README.md

```diff
@@ -0,0 +1,8 @@
+This directory is where you'll write all of your code!
```

Comment:
README does not go inside directory. It has to be at the root so that it shows up on github.